genshi/tests/input.py

```diff
@@ -278,6 +278,21 @@ def test_multibyte_character_on_chunk_boundary(self):
         self.assertEqual(1, len(events))
         self.assertEqual((Stream.TEXT, text), events[0][:2])
 
+    def test_convert_ElmentTree_to_markup_stream(self):
+        tree = et.fromstring(
+            u'<div class="test_div">text<span>some more text</span></div>'
+            )
+        ret = list(ET(tree))
```

Comment:
```suggestion
        events = list(ET(tree))
```
The other tests seem to call this list events, so lets do the same in this one.